cms/src/components/content/EditContentText.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { describe, it, expect, vi, beforeAll, afterAll } from "vitest";
+import { mount } from "@vue/test-utils";
+import { createTestingPinia } from "@pinia/testing";
+import { fullAccessToAllContentMap, mockEnglishContentDto } from "@/tests/mockData";
+import { setActivePinia } from "pinia";
+import { useUserAccessStore } from "@/stores/userAccess";
+import { ref } from "vue";
+import type { ContentDto } from "@/types";
+import EditContentText from "./EditContentText.vue";
+
+describe("EditContentPreview.vue", () => {
+    beforeAll(async () => {
+        setActivePinia(createTestingPinia());
+
+        const userAccessStore = useUserAccessStore();
+        userAccessStore.accessMap = fullAccessToAllContentMap;
+    });
+
+    afterAll(() => {
+        vi.clearAllMocks();
+    });
+
+    it("does not display text when not defined", async () => {
+        const content = ref<ContentDto>(mockEnglishContentDto);
+        const wrapper = mount(EditContentText, {
+            props: {
+                disabled: false,
+                content: content.value,
+            },
+        });
+
+        expect(wrapper.html()).not.toContain(content.value.text);
+    });
```

Comment:
This test is not testing what the description says. mockEnglishContentDto has it's "text" property set, so the text area should show. The mock data should be modified so that the text field is removed, and then check if the div containing the text editor is not rendered.